test/e2e-common.sh

```diff
@@ -26,7 +26,7 @@ function install_pipeline_crd() {
   cat "${ko_target}" | sed -e 's%"level": "info"%"level": "debug"%' \
       | sed -e 's%loglevel.controller: "info"%loglevel.controller: "debug"%' \
       | sed -e 's%loglevel.webhook: "info"%loglevel.webhook: "debug"%' \
-      | kubectl apply -R -f - || fail_test "Build pipeline installation failed"
+      | kubectl apply -R -f --server-side - || fail_test "Build pipeline installation failed"
```

Comment:
Added `--server-side` to (hopefully) fix the `metadata.annotations: Too long: must have at most 262144 bytes` error that was occurring for all e2e tests. I was also seeing this locally. `--force-conflicts` may be needed as well or possibly using `kubectl create` instead of `apply`?